src/main/java/org/candlepin/config/ConfigProperties.java

```diff
@@ -149,6 +149,9 @@ private ConfigProperties() {
     // Messaging
     public static final String CPM_PROVIDER = "candlepin.messaging.provider";
 
+    // OCSP stapling
+    public static final String SSL_VERIFY = "candlepin.sslverifystatus";
```

Comment:
Let's also set the default value (false) lower down in the same class as we do with other properties

Reply:
Done

Reply:
Why are we defaulting it to false if this is just a flag that has no barring on CP itself? From how this has been described to me multiple times now, it sounds like it's going to be available more than it's not, and it's up to each individual host to decide to shut it off if they don't/can't support it. If we default to false, every existing Sat will need to explicitly add the entry to turn it on.

Not gonna lie, this whole thing seems real hacky and weird; I'm not a fan. It feels like CP should be at least checking something locally with the stack to check if the feature is something we can support even if the config is set to true.

Reply:
@Ceiu It's the opposite: Satellite as of now does not use OCSP, so has no need for this config/capability. It is only Hosted that will roll out OCSP, and they can easily set this to true explicitly at that moment.

Unfortunately candlepin can't actually check anything, because OCSP is not provided by the candlepin/tomcat stack. It's provided by the CDN. To the extent I was involved in the design for this, I've not heard of a better solution to this.

Reply:
ack; disregard my comment